doc(v1.9.0): update notes for sle micro in longhorn command line interface installation (#1077)
longhorn/website: content/docs/1.9.0/deploy/install/_index.md

```

> **Note**:

> > Some immutable Linux distributions, such as SUSE Linux Enterprise Micro (SLE Micro), require you to reboot worker nodes after running the `install` sub-command. After the reboot, you will have to run the `install` sub-command again to complete the operation.

Reviewer: ```suggestion
> Some immutable Linux distributions, such as SUSE Linux Enterprise Micro (SLE Micro), require you to reboot worker nodes after running the `install` sub-command. After the reboot, you must run the `install` sub-command again to complete the operation.
```

Author: Addressed, thanks.
